### PR TITLE
Add argument to optionally create environment when parsing conf files

### PR DESCRIPTION
**Pull request by Haichao-Zhang**
The option is useful in the cases when we only want to parse conf files but does not want to (re-)create environments and/or set random seeds.

For example, this can be used in the async mode of hobot project, where we want to parse the confs but does not want to touch anything related to environments.

### CODE REVIEW

**Comment by emailweixu** [comment]
Why is this necessary?

**Comment by Haichao-Zhang** [comment]
> Why is this necessary?

The option is useful in the cases when we only want to parse conf files but does not want to (re-)create environments and/or set random seeds.

For example, this can be used in the async mode of hobot project, where we want to parse the confs but does not want to touch anything related to environments.

File: alf/config_helpers.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def adjust_config_by_multi_process_divider(ddp_rank: int,
         config1('TrainerConfig.evaluate', False, raise_if_used=False)
 
 
-def parse_config(conf_file, conf_params):
+def parse_config(conf_file, conf_params, set_random_seed=True):
```
Comment (emailweixu) [inline alf/config_helpers.py]: It seems to me create_env is better than set_random_seed.

Comment (Haichao-Zhang) [reply alf/config_helpers.py]: Changed argname to create_env

**Review by emailweixu — APPROVED** [review]

**Pull request merged**